CHANGELOG.md

```diff
@@ -7,13 +7,13 @@ v1.0.0
 ------
 
 ## What's Changed
-* bugfix/44-rna-test-failing by @DJTechnoo in https://github.com/highcharts/connectors-morningstar/pull/48
-* docs/docs-titles by @kamil-musialowski in https://github.com/highcharts/connectors-morningstar/pull/50
-* fetch-demo by @DJTechnoo in https://github.com/highcharts/connectors-morningstar/pull/53
-* Corrected URLs. by @bm64 in https://github.com/highcharts/connectors-morningstar/pull/47
-* Added security details docs by @bm64 in https://github.com/highcharts/connectors-morningstar/pull/49
-* Feature/investment screener by @pawellysy in https://github.com/highcharts/connectors-morningstar/pull/52
-* Added Morningstar to HC namespace. by @kamil-musialowski in https://github.com/highcharts/connectors-morningstar/pull/46
+* Fixed failing RNANews test, by @DJTechnoo in https://github.com/highcharts/connectors-morningstar/pull/48
+* Adjusted docs titles, by @kamil-musialowski in https://github.com/highcharts/connectors-morningstar/pull/50
+* Created manual fetch demo, by @DJTechnoo in https://github.com/highcharts/connectors-morningstar/pull/53
+* Corrected URLs, by @bm64 in https://github.com/highcharts/connectors-morningstar/pull/47
+* Added security details docs, by @bm64 in https://github.com/highcharts/connectors-morningstar/pull/49
+* Added investment screener, by @pawellysy in https://github.com/highcharts/connectors-morningstar/pull/52
+* Added Morningstar to HC namespace, by @kamil-musialowski in https://github.com/highcharts/connectors-morningstar/pull/46
```

Comment:
Do we want to leave those `@username`?

Reply:
We can keep them all, but improve them once we get good automation in place